docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3.6'
```

Comment:
why?

Reply:
because otherwise my docker (version 18.03.1-ce) is not working

Reply:
upgrade your docker version :P

Reply:
that's unfair, we should support as many versions we can.

Reply:
we should support a version since 2018-04-26????

rif: https://docs.docker.com/engine/release-notes/18.03/#18031-ce

Reply:
Now we can push version to 3.8, right? 😜

Reply:
for me no problem! :P